### ISSUE TITLE
[PRE REVIEW]: tradestatistics: Open Trade Statistics API Wrapper and Utility Program

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1696]
**Submitting author:** @pachamaltese (<a href="http://orcid.org/0000-0003-1017-7574">Mauricio Vargas</a>)
**Repository:** <a href="https://github.com/ropensci/tradestatistics" target ="_blank">https://github.com/ropensci/tradestatistics</a>
**Version:** 0.2.3
**Editor:** Pending
**Reviewer:** Pending

**Author instructions**

Thanks for submitting your paper to JOSS @pachamaltese. **Currently, there isn't an JOSS editor assigned** to your paper.

@pachamaltese if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```


**What happens now?**

This submission is currently in a `pre-review` state which means we are waiting for an editor to be assigned and for them to find some reviewers for your submission. This may take anything between a few hours to a couple of weeks. Thanks for your patience :smile_cat:

You can help the editor by looking at [this list of potential reviewers](https://bit.ly/joss-reviewers) to identify individuals who might be able to review your submission (please start at the bottom of the list). Also, feel free to suggest individuals who are not on this list by mentioning their GitHub handles here.

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01696/joss.01696/10.21105.joss.01696.pdf)

**Comment by pachamaltese** [comment]
potential reviewer @noamross

**Comment by pachamaltese** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01696/joss.01696/10.21105.joss.01696.pdf)

**Comment by pachamaltese** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01696/joss.01696/10.21105.joss.01696.pdf)

**Comment by arfon** [comment]
:wave: @pachamaltese - this submission was deemed ineligible to be published in JOSS back in https://github.com/openjournals/joss-reviews/issues/1366, could you explain the resubmission?

**Comment by pachamaltese** [comment]
Right. The previous submission had a text focused just in a part of the
project. After reading about citesdb, I realize I could extend more about
the API and the steps I had to complete in order to ease trade data access
in my region. That's why I don't consider this as minor utility.

**Comment by Kevin-Mattheus-Moerman** [comment]
@pachamaltese can you please elaborate a list the specific changes you made since #1366, please list:

# Changes made to the software?
- What improvements were made? 
- What new functionality/capabilities were added? 

# Changes made to the paper
- What updates were made? New content? Improved descriptions? Did you add references? etc.

**Comment by pachamaltese** [comment]
Hi Kevin

Sure !!

# Changes made to the software?

* It is peer reviewed software (rOpenSci), so it remains stable with some changes to ease using from time to time

# Changes made to the paper

* At the risk of sounding "too sociological", I added references to Open Data, and in particular to adoption and barriers of open data
* I briefly described a latin-american context problem that is basically that the universities here are broken. I try to explain that my project is not just an API wrapper, the most important part was to annoy people by email until they realised that in this region access to articles/data is not granted, and I convinced them to give me a temporary access to create a server from scratch in order to clean and organize data that is freely available for any individual/ngo/etc as long as they use it for non-commercial purposes (bc I'm using COMTRADE data which has some re-distribution clauses)
* In the end, trade statistics is a productive version of "old man yells at cloud", and I tried to move from "I'm angry bc I cannot access this data" to "I made the data available and in a legal way"

That's it, I feared that the article could sound a bit political, but anyways, this pro-open data position is already a geeky political position.
